tools/update_flutter_engine

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# 
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+base_dir=$(dirname "$0")
```

Comment:
Put `$()` in quotes.

Reply:
This is why I'm rewriting in Dart 😁 

Done.